mods/taskbar-empty-space-clicks.wh.cpp

```diff
@@ -1754,12 +1754,22 @@ void OpenTaskManager(HWND taskbarhWnd)
 {
     LOG_TRACE();
 
-    LOG_INFO(L"Sending OpenTaskManager message");
-    // https://www.codeproject.com/Articles/14380/Manipulating-The-Windows-Taskbar
-    if (SendMessage(taskbarhWnd, WM_COMMAND, MAKELONG(420, 0), 0) != 0)
-    {
-        LOG_ERROR(L"Failed to send OpenTaskManager message");
-    }
+    LOG_INFO(L"Opening Taskmgr.exe using ShellExecuteEx");
+    std::wstring taskmgrPath = L"C:\\Windows\\System32\\Taskmgr.exe";  
```

Comment:
How about something like:
```cpp
    WCHAR szWindowsDirectory[MAX_PATH];
    GetWindowsDirectory(szWindowsDirectory, ARRAYSIZE(szWindowsDirectory));
    std::wstring taskmgrPath = szWindowsDirectory;
    taskmgrPath += L"Taskmgr.exe";  
```
Or any other method to avoid hardcoding `C:\Windows`?

Reply:
I mean I get your point, but can it even ever happen that Windows directory is not located on C:\Windows?

Reply:
It's not common, but I believe it can be `D:\Windows`, not sure about anything that's not `X:\Windows`.

Reply:
> We used to use SCCM to image/reimage computers and we had some laptops with unusual drive configurations and they'd end up with the system drive being drive 'D'. Where Windows was installed and where everything else was set up. Mostly, it didn't matter, but there were two problems.
> [...]

https://www.reddit.com/r/Windows10/comments/1dlph5b/windows_on_d_drive_instead_of_c/

So seems possible but quite uncommon. Up to you, I can merge it as is.

Reply:
What a weird world we live in 🙄 Updated...